tests/nnfw_api/CMakeLists.txt

```
@@ -10,9 +10,10 @@ endif(NOT BUILD_ONERT)
 nnfw_find_package(GTest)
 
 set(RUNTIME_NNFW_API_TEST nnfw_api_gtest)
+file(GLOB_RECURSE RUNTIME_NNFW_API_TEST_LIB "lib/*.cc" "lib/*.cpp")
 file(GLOB_RECURSE RUNTIME_NNFW_API_TEST_SRC "src/*.cc" "src/*.cpp")
```

(Optional)
How about change the file names to "*.test.cc" or "*.test.cpp" if you intend it? Then, it may ensure that `src` directory has only test files in the future.
```suggestion
file(GLOB_RECURSE RUNTIME_NNFW_API_TEST_SRC "src/*.test.cc" "src/*.test.cpp")
```

---

That seems better. I'll fix soon!